private/buf/cmd/buf/command/mod/modprune/modprune.go

```diff
@@ -28,61 +28,39 @@ import (
 	"github.com/bufbuild/buf/private/pkg/rpc"
 	"github.com/bufbuild/buf/private/pkg/storage/storageos"
 	"github.com/spf13/cobra"
-	"github.com/spf13/pflag"
-)
-
-const (
-	dirFlagName = "dir"
 )
 
 // NewCommand returns a new prune Command.
 func NewCommand(
 	name string,
 	builder appflag.Builder,
 ) *appcmd.Command {
-	flags := newFlags()
 	return &appcmd.Command{
-		Use:   name,
+		Use:   name + " <input>",
```

Comment:
We should probably use `<directory>` so that it's clearly distinguished from the generic `<input>` supported by `buf build` and others.

Reply:
Oh yeah, good call out -- you're right, this should only be for local state :D